### ISSUE TITLE
Bad rendering in Dev docs installation step

### ISSUE THREAD

**Issue by AthulMuralidhar** [thepracticaldev/dev.to#6922]
**Describe the bug**

- Going through the dev docs, we see that the rendering is a bit off. The menues seem to overlay the text

**To Reproduce**

- Go to: https://docs.dev.to/installation/
- Scroll down to the various installation techniques (right after the **Gitpod** step)
- You should be here:
![Screenshot from 2020-03-28 15-43-27](https://user-images.githubusercontent.com/35686547/77825658-00c96900-710b-11ea-8315-5ea760fd451d.png)
- Now gently scroll down and you'll spot the bug like so:
![Screenshot from 2020-03-28 15-45-29](https://user-images.githubusercontent.com/35686547/77825684-2e161700-710b-11ea-857d-d6d1dbd2b248.png)

**Expected behavior**

- The overlay should not be happening

**Screenshots**
- Provided above

**Desktop (please complete the following information):**

- OS: Ubuntu 19.10 
- Browser: Mozilla Firefox 74.0 and Google Chrome 80.0.3987.149

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Interesting, I cannot reproduce it with Firefox 75.0b7 on macOS 10.14.6:

![docs](https://user-images.githubusercontent.com/146201/77830595-d509ab80-7129-11ea-9947-2e2237159a8a.gif)

same with Chrome

**Comment by rhymes** [comment]
Never mind, I was able to reproduced when I resized the window :(

**Comment by vabelha** [comment]
I messed around a bit with this bug and the problem is related with the style applied to the div markdown and only happens when it maxes out - reaches 850px.
```css
max-width:850px;
```
A fix would be just adding overflow-hidden or a media-query to reduce max-width.
Yet, I spent some time searching and if I am not mistaken gitdocs does not support custom styling.

**Comment by rhymes** [comment]
@vabelha apparently not https://github.com/timberio/gitdocs/issues/83

**Comment by vabelha** [comment]
Yeah I saw it. It seems though gitdocs isn't actively maintained. 
I thought I could add a style to override the existing style in the file api_template.hbs but I guess that it serves no purpose in the generation of docs. 

I just ended up creating a PR with a possible fix: timberio/gitdocs/pull/184

**Comment by rhymes** [comment]
Awesome!

**Comment by vabelha** [comment]
Another docs page that comes to my mind is https://docusaurus.io/

**Comment by Amorpheuz** [comment]
I would love to see docs updated to a more feature-rich doc framework too! Apart from Docusaurus, there is even https://github.com/docsifyjs/docsify/ and https://github.com/mkdocs/mkdocs/ too.
